src/protocols/soroban/index.ts

```diff
@@ -0,0 +1,133 @@
+import * as naj from "near-api-js"
```

Comment:
this is `soroban/index`?

Reply:
Yeah it's all throughout. I just wanted to get to the point where the frontend would print out the schema.